monkey/monkey_island/cc/setup/plugin_installation.py

```diff
@@ -0,0 +1,18 @@
+from pathlib import Path
+
+from common import DIContainer
+from monkey_island.cc.services.agent_plugin_service import IAgentPluginService
+
+from .data_dir import PLUGIN_DIR_NAME
+
+
+def install_plugins(container: DIContainer, data_dir: Path) -> Path:
+    agent_plugin_service = container.resolve(IAgentPluginService)
+
+    plugins_dir = data_dir / PLUGIN_DIR_NAME
+    plugin_tar_file_paths = list(plugins_dir.glob("*.tar"))
+
+    for path in plugin_tar_file_paths:
+        with open(path, "rb") as f:
```

Comment:
```suggestion
        if path.is_symlink():
            #Log warning and skip this. It's unsafe.
        if not path.is_file():
            # Log warning and skip this, we only handle plugin files
        with open(path, "rb") as f:
```

Reply:
I don't think we need this either?
https://github.com/guardicore/monkey/pull/3603#discussion_r1302581671

Reply:
Just because something is named ".tar" doesn't mean it actually is a tar file.